### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by woodshop**
Fix types that prevent from running.

### CODE REVIEW

File: wavenet.py [hunk 1]
```diff
@@ -94,7 +94,7 @@ def _create_network(self, input_batch):
 
             # We skip connections from the outputs of each layer, adding them
             # all up here.
-            total = sum(out)
+            total = sum(outputs[1:])
```
Comment (ibab) [inline wavenet.py]: Why did you leave out `outputs[0]` here?

**Comment by ibab** [comment]
Thanks for fixing these!
I just have a question on one of the lines (see above)

**Comment by woodshop** [comment]
I did not mean to remove `outputs[0]` in the summation. I've fixed the error. I also noticed another type where an unknown variable `i` was being used instead of `step`. I've made these changes and tried to squash into less commits.

File: wavenet.py (hunk 1 above)
Comment (lemonzi) [inline wavenet.py]: I think this should be `sum(outputs)`, I mis-used a temporary variable.

**Comment by lemonzi** [comment]
These were my fault, sorry about that! I was about to send a patch.

**Comment by ibab** [comment]
This looks good! Thank you!
@lemonzi: No worries. I should set up automated testing.

**Pull request merged**